ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -1351,34 +1272,6 @@ describe(RuntimeConfigurationPanel.name, () => {
     ).not.toBeInTheDocument();
   });
 
-  it('should allow Dataproc -> PD transition', async () => {
```

Comment:
is there an equivalent for this one?

Reply:
Added 'should allow Dataproc -> PD transition' to runtime-hooks test file.